internal/actions/object/user_grant.go

```diff
@@ -58,10 +61,11 @@ func AppendGrantFunc(userGrants *UserGrants) func(c *actions.FieldConfig) func(c
 	}
 }
 
-func UserGrantsFromQuery(c *actions.FieldConfig, userGrants *query.UserGrants) goja.Value {
+func UserGrantsFromQuery(ctx context.Context, queries *query.Queries, c *actions.FieldConfig, userGrants *query.UserGrants) goja.Value {
```

Comment:
Cloudn't the UserGrantsFromQuery and UserGrantsFromSlice be combined?

Reply:
As we're optimizing the token endpoint, the code redundancy will be removed anyway soon.

Reply:
Ok, works for me